change/@ni-nimble-components-0911f623-6d16-4c80-9b6d-0757470b2ca7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We probably need some docs for aggregate bindings not being allowed to use any conditional logic right?

Reply:
I don't mind adding docs. I hadn't because @jattasNI [commented on the issue](https://github.com/ni/nimble/issues/1843#issuecomment-2065090524) saying that he wasn't sure they were needed. Do you think we should add something to the contributing docs about this?

Reply:
Yea guess it's weird to document. I think both of those behaviors could reasonably considered bugs.
I at least captured it in the PR review common issues, guess we can just do that for now: https://github.com/ni/nimble/issues/1479#issuecomment-2066815162